.nvmrc

```diff
@@ -1 +1 @@
-v16
+lts/*
```

Comment:
Adding to match module template
[.nvmrc](https://github.com/MetaMask/metamask-module-template/blob/354995b0a4040cbb74f07b42a0fbc279fb5a1ea0/.nvmrc